release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
+        {
+            "channel-version": "2.1",
+            "latest": "2.1.4",
+            "release-date":"2018-09-11",
+            "lts": true,
+            "eol": false,
+            "eol-date": null,
+            "releases.json": "https://github.com/dotnet/core/blob/master/release-notes/2.1/releases.json",
+            "lifecycle-policy": "https://www.microsoft.com/net/support/policy"
+        },
+        {
+            "channel-version": "2.0",
+            "latest": "2.0.9",
+            "release-date":"2018-05-21",
+            "lts": false,
+            "eol": false,
```

Comment:
This should be set to true now and get propagated to the 2.0 file.

Reply:
Fixed. Also updated releases-index.json.